docs/modules/envoy_access_log_server.md

```diff
@@ -0,0 +1,132 @@
+# Envoy access log server
+
+|                |                        |
+|----------------|------------------------|
+| `moduleName`   | `envoyAccessLogServer` |
+| Module type    | `producer`             |
+| Output event   | `raw`                  |
+
+This module starts a grpcServer ready to accept envoy access logs (see below for supported API version)	|
+
+### Envoy support and configuration
+At this moment, V3 of envoy's xDS API is supported. [See the upstream documentation for details](https://www.envoyproxy.io/docs/envoy/latest/api/api_supported_versions) on API versions.
+
+In particular, you have to configure your envoy instance to send access_logs using [v3 AccessLogService rpc](https://github.com/envoyproxy/envoy/blob/842485709d651a6057b2ffb505ffced21173e004/api/envoy/service/accesslog/v3/als.proto#L39). We currently do not implement handling of other versions.
+
```

Comment:
```suggestion

> Please note that Envoy [does not buffer any of the messages sent](https://godoc.org/github.com/envoyproxy/go-control-plane/envoy/service/accesslog/v3) so there may be possible los of the log messages.

```

Reply:
I would say this is misleading. Envoy allows buffering to be configured.

[doc](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/access_loggers/grpc/v3/als.proto#envoy-v3-api-msg-extensions-access-loggers-grpc-v3-commongrpcaccesslogconfig)